### PR TITLE
Double the speed when LEFT_SHIFT is pressed

### PR DESCRIPTION
**Pull request by RkShaRkz**
This was something I forgot to do in the previous PR, but was also on dark's wishlist.

### CODE REVIEW

**Review by RkShaRkz — COMMENTED** [review]
LGTM

File: source/dagonBackend.d [hunk 1]
```diff
@@ -1403,7 +1404,8 @@ final class SacScene: Scene{
 			if(eventManager.keyPressed[KEY_I]) speed = 10.0f;
 			if(eventManager.keyPressed[KEY_O]) speed = 100.0f;
 			if(eventManager.keyPressed[KEY_P]) speed = 1000.0f;
-			if(dir.lengthsqr>0.001f^^2) camera.position += dir.normalized * speed * (dt.total!"hnsecs"*1e-7);
+			if(eventManager.keyPressed[KEY_LSHIFT]) factor *= 2;
```
Comment (tg-2) [inline source/dagonBackend.d]: You can declare `factor` in front of this line to reduce its scope.

Comment (tg-2) [reply source/dagonBackend.d]: Should be `factor *= 2.0f`. I adhere to this convention to make it a bit more clear where the floating-point operations are.

Comment (RkShaRkz) [reply source/dagonBackend.d]: alright, fair point.

Comment (RkShaRkz) [reply source/dagonBackend.d]: done.

**Pull request merged**